src/app/shared/helper-services/update-data.service.ts

```diff
@@ -56,6 +56,10 @@ export class UpdateDataService {
             assessment.fsat.fieldData.inletVelocityPressure = 0;
             assessment.fsat.fieldData.usingStaticPressure = true;
         }
+
+        if(assessment.fsat.fieldData['specificHeatRatio']) {
```

Comment:
Lets add  `&& !assessment.fsat.baseGasDensity.specificHeatRatio`
 to the check..

Also should be updating modifications?